typescript-forms/form-values.ts

```diff
@@ -0,0 +1,21 @@
+interface FormElements extends HTMLFormControlsCollection {
+  name: HTMLInputElement;
+  email: HTMLInputElement;
+  message: HTMLTextAreaElement;
+}
+
+const $contactForm = document.querySelector('#contact-form') as HTMLFormElement;
+if (!$contactForm) throw new Error('#contact-form query failed');
+
+$contactForm.addEventListener('submit', (event: Event) => {
+  event.preventDefault();
+  const $formElements = $contactForm.elements as FormElements;
+
+  const obj: object = {
+    name: $formElements.name.value,
+    email: $formElements.email.value,
```

Comment:
You don't need this type `object` here and I would advise against using type `object`. 

If you need to type an object, either use a `Record` type if the number of properties is variable, is an `interface` to make more specific typing, or use _type inference_ where TypeScript will _infer_ what the object will be made of based on what is assigned to it.